python/tank/descriptor/io_descriptor/base.py

```diff
@@ -570,6 +621,18 @@ def _get_primary_cache_path(self):
         """
         return self._get_bundle_cache_path(self._bundle_cache_root)
 
+    def _is_settings_folder(self, path):
+        """
+        Returns if a path is matching the pattern for a settings folder
+        """
+        return path.endswith(".settings")
```

Comment:
I am not a fan of .settings TBH. I think people might confuse this with bundle's settings. Maybe `.metadata` would be safer?